src/main/java/io/jenkins/plugins/synopsys/security/scan/global/Utility.java

```diff
@@ -158,4 +177,23 @@ private static String getMaskedProxyUrl(String proxyUrlString) throws MalformedU
 
         return proxyUrlString;
     }
+
+    public static int compareVersions(String version1, String version2) {
+        int comparisonResult = 0;
```

Comment:
We can use ComparableVersion class for version comparison.

Reply:
Initially I intended to use the `ComparableVersion` class but to use this class we need to add `maven-artifact` dependency in the `pom.xml`
After adding the `maven-artifact` dependency, when I tried to run the plugin there were some version conflicting with github-branch-source plugin. I used 3.9.6 but to resolve the conflicts with `github-branch-source` it needs the latest version of `maven-artifact`. But the issue is the latest version is an _alpha_ version, so I was not sure whether we should use an _alpha_ version or not. 

Versions of `maven-artifact` are here: https://mvnrepository.com/artifact/org.apache.maven/maven-artifact

Also I have not included the _alpha_ version thinking if some vulnerability comes up in _alpha_ version. These are the reason I have chosen to write an extra method instead of adding one more dependency (I have noticed that this implementation is widely used).

@kishorikumar let me know your thoughts on this.

Reply:
Updated as per your recommendation (with `Version` class of jackson core library).

Changes can be found here - https://github.com/jenkinsci/synopsys-security-scan-plugin/pull/47/commits/3a85d248d6c40a29904eafe082fc6f7112bdaf4a#diff-9b3e71adffe48a65a85abffce484b57bd1b70e8979a60bc3b2a1419afbd75c49R178